src/Aether/Aether.fs

```diff
@@ -110,39 +110,39 @@ module Compose =
 
 (* Lenses
 
-   Various lenses implemented for common types such as tuples,
-   lists and maps, along with an id lens (which is useful for composing
-   a lens which has not specific "lensing" elements but is implicitly a chain
-   of one or more isomorphisms. Having an id lens enables the root composition. *)
+   Various lenses for common types such as tuples,
+   lists, and maps; along with an id_ lens (which is useful for composing
```

Comment:
I think you're right that identity would be better. It does have a well specified meaning. No problem, I know that the lens stuff in general is quite a big change for people new to FP. I really appreciate you getting involved, it's helpful to have people viewing these things from different perspectives.

The documentation is going to get a distinct overhaul soon, as we could do a much better job about introducing "specialist" terminology and giving a better intro to why the whole concept is useful.